src/applications/proxy-rewrite/README.md

```diff
@@ -25,3 +25,34 @@ http://localhost:3001/?target=https://www.va.gov/health/
 
 ## Charles Proxy
 You can also use an application called Charles Proxy to map the `proxy-rewrite` bundles of TeamSite pages to your local machine. This way you can navigate directly to `https://www.va.gov/health/` and when the request for the production bundle of `proxy-rewrite` is sent, Charles will have overridden that file to instead be served locally. Instructions to set this up are located here, https://github.com/department-of-veterans-affairs/vets.gov-team/blob/master/Work%20Practices/Engineering/Teamsite.md.
+
+## Teamsite Visual Regression Testing
+
+Both scripts compare production Teamsite header and footer against images in `vets-website` repo
```

Comment:
Would like to move the baseline images out of this repo. At first I was thinking we could use `S3` but I think using a separate github repo might be better:
- you wouldn't need access to amazon if you needed to do something manual to the images
- github tracks history of files automatically (easier to restore)
- we could use Lerna to manage it as a sub repo

The diff images are a different story- I'm going to explore what we can push to github from Jenkins when I automate this. Open to ideas here. The diff images are just a flat list of images with a long file name that includes the entire name of the jest test- I would like to associate failures to a ticket so they're easy for engineers to review.

Reply:
I think Github for both sets of images could work. We could save them for each run. We can definitely update GH from Jenkins, so I'm pretty sure it's possible. But I think we should talk with devops to see if they have other suggestions.